src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp

```diff
@@ -536,18 +553,17 @@ namespace OpenMS
     defaults_.setValue("isotopic_similarity", "false", "Computes a similarity score for each hit (only if the feature exhibits at least two isotopic mass traces).");
     defaults_.setValidStrings("isotopic_similarity", {"false", "true"});
 
-    defaults_.setValue("db:mapping", std::vector<std::string>{"CHEMISTRY/HMDBMappingFile.tsv"}, "Database input file(s), containing three tab-separated columns of mass, formula, identifier. "
+    //TODO should we really support multiple databases? Not sure
+    defaults_.setValue("db:mapping", std::vector<std::string>{"CHEMISTRY/HMDB.sqlite"}, "Database input file(s), containing three tab-separated columns of mass, formula, identifier. "
```

Comment:
```suggestion
    defaults_.setValue("db:mapping", std::vector<std::string>{"CHEMISTRY/HMDB.sqlite"}, "Database input file(s), containing a compounds sqlite table with six columns of Mz, Id, Formula, Name, SMILES, INCHI. "
```

Reply:
does the table need a specific name?

Reply:
exactly. I will put quotes around 'compounds' in your doc addition